File: frontends/c-sh-go/testdata/t07_cmp.c
#include <stdio.h>
int main(void) {
    int a = 5;
    int b = 5;
    if (a == b) { printf("eq\n"); }
    if (a != 3) { printf("ne\n"); }
    if (a >= 5 && b <= 5) { printf("both\n"); }
    return 0;
}

File: frontends/rust-frontend/testdata/t07_cmp.rs
fn main() {
    let a = 5;
    let b = 5;
    if a == b { println!("eq"); }
    if a != 3 { println!("ne"); }
    if a >= 5 && b <= 5 { println!("both"); }
    if a > 2 || b < 1 { println!("or"); }
    if !(a == 1) { println!("not"); }
}
